src/_guides/language/effective-dart/documentation.md

```diff
@@ -376,10 +382,10 @@ class ToggleComponent {}
 ## Markdown
 
 You are allowed to use most [markdown][] formatting in your doc comments and
-dartdoc will process it accordingly using the [markdown package][].
+dartdoc will process it accordingly using the [markdown package.][]
 
 [markdown]: https://daringfireball.net/projects/markdown/
-[markdown package]: https://pub.dartlang.org/packages/markdown
+[markdown package.]: https://pub.dartlang.org/packages/markdown
```

Comment:
I find it slightly odd that the period is part of the link, but maybe that's accepted style?

Reply:
Yeah, it's the convention on our site. It's because it looks weird when the external link indicator forces a space between the word and the punctuation that follows it.